com.zsmartsystems.zigbee/src/main/java/com/zsmartsystems/zigbee/zcl/ZclCluster.java

```diff
@@ -490,7 +493,10 @@ public boolean getApsSecurityRequired() {
         command.setDstAddrMode(3); // 64 bit addressing
         command.setDstAddress(address);
         command.setDstEndpoint(endpointId);
-        return zigbeeEndpoint.sendTransaction(command, new UnbindRequest());
+        // The transaction is not sent to the Endpoint of this cluster, but to the ZDO endpoint 0 directly.
+        // Hence calling here zigbeeEndpoint.getParentNode().sendTransaction() instead of
```

Comment:
Can I ask that you remove the second and third lines of these two comments? ie I think the first line explains the point, and the code shows what is called ;)

Reply:
Done :)

Reply:
Many thanks Pedro - I'll merge once CI completes.